lib/crowbar/init/application.rb

```diff
@@ -34,7 +34,9 @@ class Application < Sinatra::Base
       before do
         logger.level = Logger::DEBUG
         # parse JSON in POST requests
-        if request.request_method == "POST" && request.content_type.include?("json")
+        if request.request_method == "POST" && (
+            !request.content_type.nil? &&
+            request.content_type.include?("json"))
```

Comment:
wow hound stopped complaining :) but maybe we can do it like this for readability?
```ruby
if request.request_method == "POST" &&
    (!request.content_type.nil? && request.content_type.include?("json"))
```